posts/how-to-commit-code-to-a-git-repository-with-visual-studio-code.md

```diff
@@ -0,0 +1,152 @@
+---
+title: How to commit code to a Git repository with Visual Studio Code
+author: marek-zaluski
+publicationDate: September 9, 2021
+tags: [tutorial, git, IDE, open source]
+description: Learn how to create commits in a Git repository with VS Code.
+image: https://storage.googleapis.com/sourcegraph-assets/learn/headers/sourcegraph-learn-02.png
+imageAlt: Sourcegraph Learn
+type: posts
+---
+
+Committing code with Git is useful whether you’re working on your own projects, collaborating with others, or contributing to an open source project. Visual Studio Code is an open source IDE (Integrated Development Environment) with support for many programming languages. In addition to being a capable tool for writing and editing source code, VS Code can help you interact with Git repositories and commit changes. If you’re already using VS Code to edit code, it’s convenient to use the same tool to commit your changes with Git.
+
+In this tutorial, we’ll demonstrate how to use the Git features within VS Code to commit code to a Git repository.
+
+## Prerequisites
+
+To follow this tutorial, you need Visual Studio Code installed on your computer. You can install it by following the instructions on the official [Visual Studio Code website](https://code.visualstudio.com/).
+
+You also need Git installed. Visual Studio Code’s Git features depend on the Git command-line tools. You can install Git by following the instructions on the [“Installing Git” page of the official documentation](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+With the above software installed, you should have created or identified a Git repository that you would like to contribute to. To make one of your local code repositories a Git repository, you can review the official documentation for “[Getting a Git Repository.](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository)”
```

Comment:
The full stop should be out of the link here. Wondering why this changed from the last PR, does this document need a full new review?

Reply:
It must have been an unintended mix-up while fixing the other link. It's fixed.